src/data/markdown/docs/01 guides/05 Testing Guides/02 Load testing websites.md

```diff
@@ -3,115 +3,99 @@ title: "Load testing websites"
+But this is not necessarily accurate. 
 
-The first thing you have to decide is what to load test first.
+When the number of users visiting your website increases, the frontend time will always be the same, but if your system struggles with the increased concurrency, the **backend time might grow exponentially with the rise of concurrent users**.
 
-On one side, you should evaluate which are the performance aspects that are most valuable to your business and has the most significant risks.
+The following chart illustrates this situation:
 
-On the other side, you have to identify what your users will do regularly. You could look at your analytics tools to find the most frequent user journeys.
+![](./Frontend-Backend-LoadTesting.png)
 
-With this information, it’s time to analyze the frequency of usage, business value, performance risks, and any other critical performance aspect of your organization to decide what to load test first.
 
-As a rule of thumb, <a href="https://k6.io/our-beliefs#simple-testing-is-better-than-no-testing">Simple testing is better than no testing</a>:
+## When should I load test a website?
 
-> Start small and simple, make sure you get something out of the testing first, then expand the test suite and add more complexity until you feel that you’ve reached the point where more effort spent on realism will not give enough return on your invested time.
+In short, when you are concerned about the **availability and scalability of your website**. 
 
+If you have very low number of users, backend performance won't likely be an issue. Spend your time optimizing the frontend. 
 
-Once you adopt performance testing in your development lifecycle, your performance test suite will grow over time. You will add and update performance tests based on the evolution of your application, new performance areas, bugs, and the maturity of your team and processes, amongst other factors.
+But as the number of user grows, you should start spending more time improving and testing the performance of your backend. In the [performance golden rule](https://www.stevesouders.com/blog/2012/02/10/the-performance-golden-rule/), Steve Souders pointed out that:
 
-To sum up, start small, test frequently and iterate, iterate, iterate.
+> If you’re worried about availability and scalability, focus on the backend.
 
-## Should I load test my website in production or pre-production?
+### Browser metrics and load testing
 
-We recommend to <a href="https://k6.io/our-beliefs#load-test-in-a-pre-production-environment">Load test in a pre-production environment</a> that mimics the production environment as closely as possible.
+When testing how your system behaves with some concurrent users, browser metrics are, in most cases, less useful. The reason is that each browser runs independently of the number of users visiting your website.  Hence, a load test will not impact browser metrics like rendering time.
```

Comment:
When testing how your system behaves with some concurrent users, browser metrics are, in most cases, less useful. 

Because each browser runs independently, the number of concurrent users in a load test will not affect browser metrics like rendering time.